pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["setuptools", "wheel", "versioneer"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "MembraneCurvature"
+version = "1.0.0"
+license = {file = "LICENSE"}
+description = "MDAnalysis tool to calculate membrane curvature from MD simulations."
+authors = [
+    {name = "Estefania Barreto-Ojeda", email = "estefania@ojeda-e.com"},
```

Comment:
Not sure if relevant. Here in `authors`, should I add more names from the MDA core devs?

Reply:
Your name should definitely be there. If there are other major contributors you can add them. You might be able to reference a AUTHORS file — check the syntax for pyproject.toml.

Reply:
Thnaks @orbeckst 
I added the AUTHORS file, however, the `toml` file doesn't like the syntax `authors = { file = "AUTHORS" }`. I checked and the wheel is not generated:

```
      {"file":"AUTHORS"} is not of type "array"
      ValueError: invalid pyproject.toml config: `project.authors`.
      configuration error: `project.authors` must be array
```
Can't find any example that parses a file. According to [setuptools docs](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration), the AUTHORS file is added by default in the `license-files` key.  I checked and it is the file is added to the wheel. Here ta snippet of the wheel contents:
```
      292  01-02-2024 22:50   membrane_curvature-1.0.0+16.g5d02d31.dist-info/AUTHORS
    34684  01-02-2024 22:50   membrane_curvature-1.0.0+16.g5d02d31.dist-info/LICENSE
    47817  01-02-2024 22:50   membrane_curvature-1.0.0+16.g5d02d31.dist-info/METADATA
```

Reply:
Yeah as far as I'm aware authors can be a list of dictionaries but not a file. Personally, I would stick with yourself here. It's good enough to have an `AUTHORS` file.